src/components/progress-tracker/progress-tracker.component.tsx

```diff
@@ -45,48 +47,44 @@ const ProgressTracker = ({
   error = false,
   progress = 0,
   description,
-  showDefaultLabels = false,
+  showDefaultLabels,
   currentProgressLabel,
   customValuePreposition,
   maxProgressLabel,
   labelsPosition = "top",
   labelWidth,
   ...rest
 }: ProgressTrackerProps) => {
+  if (!deprecatedShowDefaultLabelsWarnTriggered && showDefaultLabels) {
+    deprecatedShowDefaultLabelsWarnTriggered = true;
+    Logger.deprecate(
+      "The `showDefaultLabels` prop has been deprecated, default labels will now always be displayed if no custom labels are provided."
+    );
+  }
+
   const l = useLocale();
   const prefixLabels = labelsPosition !== "bottom";
 
   const renderValueLabels = () => {
-    if (!showDefaultLabels && !currentProgressLabel) {
-      return null;
-    }
+    let displayedCurrentProgressLabel, displayedMaxProgressLabel;
 
-    const label = (value?: string, defaultValue?: string) => {
-      if (value) {
-        return value;
-      }
-
-      return showDefaultLabels ? defaultValue : undefined;
-    };
-
-    const displayedCurrentProgressLabel = label(
-      currentProgressLabel,
-      `${progress}%`
-    );
-
-    const displayedMaxProgressLabel = label(maxProgressLabel, "100%");
+    if (currentProgressLabel) {
```

Comment:
Comment: I think it's worth adding at least one story that demos the permutations we can now have as a result of this code (apologies if we already have it I may have missed it), it will also be easier for UX to review etc

Reply:
There is an example of these in the custom label story, and then the default label is present in the rest 👍